app/models/patient.rb

```diff
@@ -78,11 +73,15 @@ class Patient < ApplicationRecord
             presence: { message: "is required when 'Isolation' is 'false'" },
             if: -> { !isolation }
 
-  validates :email, on: :api, email: true
+  validates :email, email: true
```

Comment:
I'm also wondering if there are records on prod that would not pass this email validation currently. The email validation performed on the frontend uses `yup` which as far as I understand it may be less strict: https://github.com/jquense/yup#stringemailmessage-string--function-schema.